src/sources/util/fake.rs

```diff
@@ -0,0 +1,162 @@
+use chrono::{prelude::Local, SecondsFormat};
```

Comment:
I feel like this, despite being small can be extracted to `lib/` crate right away, and potentially even to a published crate.

Reply:
I'm definitely open to this in the future! I'd like to get this landed for now but it would be cool to offer a `fakelogs` crate or something like that. We can explore that when we have more formats and more options.